setup.py

```diff
@@ -29,18 +30,26 @@ class PrepareStaticCommand(Command):
     )
     TARGET_DIR = "galaxy_ng/app/static/galaxy_ng"
 
-    user_options = []
+    user_options = [
+        ('force-download-ui',
+        None,
+        'Replace any existing static files with the ones downloaded from github.'),
+    ]
 
     def initialize_options(self):
-        pass
+        self.force_download_ui = False
```

Comment:
minor: since option `force-download-ui` gets mapped to var `self.force_download_ui`, is it necessary to initialize this as `self.force_download_ui = False` since the default is `None`?

Reply:
I _think_ the second argument here

```
    user_options = [
        ('force-download-ui',
        None,
        'Replace any existing static files with the ones downloaded from github.'),
    ]
 ```

defines a single character flag like `-f`, not the default value.

Reply:
yeah, it's long name, short name, description

https://dankeder.com/posts/adding-custom-commands-to-setup-py/

Reply:
makes sense